cmd/driver/install/install.go

```diff
@@ -54,8 +55,9 @@ func NewDriverInstallCmd(ctx context.Context, opt *options.Common, driver *optio
 		Common: opt,
 		Driver: driver,
 		// Defaults to downloading or building if needed
-		Download: true,
-		Compile:  true,
+		Download:    true,
+		Compile:     true,
+		HTTPHeaders: "",
```

Comment:
We have a `driverDownloadOptions` struct to hold http related things; can you put this in there?